inventory_requests/views/ModifyRequestCart.py

```diff
@@ -109,14 +134,58 @@ def put(self, request, *args, **kwargs):
     def perform_update(self, serializer):
         request_cart = self.get_object()
         get_or_not_found(User, pk=serializer.validated_data.get('owner_id'))
-        if not modify_request_cart_logic.can_approve_deny_cancel_disburse_request_cart(request_cart, 'disburse'):
+        if not modify_request_cart_logic.can_approve_disburse_request_cart(request_cart, 'disburse'):
             detail_str = 'Cannot disburse due to insufficient items' if request_cart.status == 'active' else \
```

Comment:
should it be cannot dispense? this error message can be thrown for insufficient items in disbursements or loans.